source/Plugins/Language/CPlusPlus/CPlusPlusLanguage.cpp

```diff
@@ -251,14 +251,14 @@ std::string CPlusPlusLanguage::MethodName::GetScopeQualifiedName() {
 bool CPlusPlusLanguage::IsCPPMangledName(const char *name) {
   // FIXME!! we should really run through all the known C++ Language plugins
   // and ask each one if this is a C++ mangled name
-  
+
```

Comment:
This looks like a bunch of whitespace fixing that's unrelated. Can you revert?

Reply:
Discussed and decided to leave it in place.